rust/src/address.rs

```diff
@@ -98,10 +106,10 @@ impl StakeCredential {
         }
     }
 
-    pub fn kind(&self) -> u8 {
+    pub fn kind(&self) -> StakeCredKind {
         match &self.0 {
-            StakeCredType::Key(_) => 0,
-            StakeCredType::Script(_) => 1,
+            StakeCredType::Key(_) => StakeCredKind::Key,
+            StakeCredType::Script(_) => StakeCredKind::Script,
```

Comment:
Thanks, Seba! Just to double-check In JS it will be returning the **values** from `declare export var StakeCredKind` so backward compatible, right?

Reply:
yes, backwards compatible for JS which is why the return type of the Flow definition didn't change. Not backwards compatible for Rust, but that shouldn't be an issue

Reply:
Perfect